tests/integration/targets/setup_postgresql_db/tasks/main.yml

```diff
@@ -49,15 +48,14 @@
 
 - name: remove old db config and files (debian)
   file:
-    path: '{{ loop_item }}'
+    path: '{{ item }}'
     state: absent
   ignore_errors: true
   when: ansible_os_family == "Debian"
   loop:
+  - '{{ pg_dir }}'
   - /etc/postgresql
   - /var/lib/postgresql
-  loop_control:
```

Comment:
Out of curiosity: why remove the loop control?

Reply:
i've no idea why, but it removes only the first item with loop_control and our tests fail when trying to initialize DB and the directory is not empty. This happened locally, and this change helped.